start

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+python /Project-env/Project/bin/manage.py redcap json /Input/metadata.csv 
+echo passed json
+python /Project-env/Project/bin/manage.py redcap models metadata.json -o /Project-env/Project/Project/models.py
+echo passed models
+python /Project-env/Project/bin/manage.py redcap fixture /Input/data.csv metadata.json Project
+echo passed sync
+python /Project-env/Project/bin/manage.py syncdb
+echo passed avocado
+python /Project-env/Project/bin/manage.py avocado init Project
+
+echo 'Loading data...'
+python /Project-env/Project/bin/manage.py loaddata fixtures.json
+
+echo "MODELTREES = {
```

Comment:
Remove this now that it is part of the `local_settings.py` file mentioned earlier.